relay-common/src/utils.rs

```diff
@@ -6,45 +6,110 @@ use regex::Regex;
 
 use crate::macros::impl_str_serde;
 
-/// A simple glob matcher.
-///
-/// Supported are `?` for a single char, `*` for all but a slash and
-/// `**` to match with slashes.
-#[derive(Clone)]
-pub struct Glob {
-    value: String,
-    pattern: Regex,
+/// Glob options is used to configure the behaviour underlying regex.
```

Comment:
I find this sentence difficult to understand, and without the example below I've not been able to. I suggest the following but feel free to modify it to a different one.
```suggestion
/// Glob options represent the underlying regex emulating the globs.
```